### ISSUE TITLE
Speaker Cannot Read Non-English Content Even with @lang Attribute (Accessibility Issue)

### ISSUE THREAD

**Issue by TaF-JingboSu** [edrlab/thorium-reader#3287]
When using Thorium Reader, the speaker functionality skips non-English content, even when the @lang or @xml:lang attributes are correctly applied to tag the non-English content. This behavior impacts accessibility, as users relying on the speaker feature cannot access content in languages other than English.

Please see the following code:
    `
    <p><b>Korean</b></p>
    <p>(span with @lang)<span lang="ko" xml:lang="ko">damunhwa</span>,original <span lang="ko" xml:lang="ko">&#45796;&#47928;&#54868;</span></p>
    <p>(i with @lang)<i lang="ko" xml:lang="ko">damunhwa</i>,original <i lang="ko" xml:lang="ko">&#45796;&#47928;&#54868;</i></p>
    <p>(i without @lang)<i>damunhwa</i>,original <i>&#45796;&#47928;&#54868;</i></p>
    <p><b>Chinese</b></p>
    <p>(span with @lang)<span lang="zh" xml:lang="zh">daluren</span>,original <span lang="zh" xml:lang="zh">大陆人</span></p>
    <p>“陆”字本义为高而平的地方，又指高出水面的陆地。“陆”由本义又引申为大土山</p>
    <p>(i with @lang)<i lang="zh" xml:lang="zh">daluren</i>,original <i lang="zh" xml:lang="zh">大陆人</i></p>
    <p>(i without @lang)<i>daluren</i>,original <i>大陆人</i></p>
    <p><b>Japanese</b></p>
    <p>(span with @lang)<span lang="jp" xml:lang="jp">Hokkaido</span>,original <span lang="jp" xml:lang="jp">北海道</span></p>
    <p>(i with @lang)<i lang="jp" xml:lang="jp">Hokkaido</i>,original <i lang="jp" xml:lang="jp">北海道</i></p>
    <p>(i without @lang)<i>Hokkaido</i>,original <i>北海道</i></p>
`
